server/src/server/statistics/KeyspaceStatistics.java

```diff
@@ -57,6 +61,15 @@ public long count(TransactionOLTP tx, Label label) {
     public void commit(TransactionOLTP tx, UncomittedStatisticsDelta statisticsDelta) {
         HashMap<Label, Long> deltaMap = statisticsDelta.instanceDeltas();
 
+        // precompute the delta for all Thing's and update the delta map
+        long thingDelta = deltaMap.values().stream()
+                .reduce((a,b) -> a+b)
+                .orElse(0L);
+        Label thingLabel = Schema.MetaSchema.THING.getLabel();
+        // it shouldn't exist but just in case we insert it and do an update
+        deltaMap.putIfAbsent(thingLabel, 0L);
+        deltaMap.put(thingLabel, deltaMap.get(thingLabel) + thingDelta);
```

Comment:
Why double put? Also, put does an overwrite if an entry already exists.

Reply:
The second put does a get first :) so we do an update in the second, line, and initialise a value on the first line